src/servers/src/http/handler.rs

```diff
@@ -138,7 +138,19 @@ pub async fn sql(
     if let Some(limit) = query_params.limit {
         resp = resp.with_limit(limit);
     }
-    resp.with_execution_time(start.elapsed().as_millis() as u64)
+
+    let cost = start.elapsed();
+    if let Some(threshold) = state.slow_query_threshold {
```

Comment:
What about move it into 

https://github.com/GreptimeTeam/greptimedb/blob/36b1bafbf00152cc89c8d4362ace432bbb1932a3/src/operator/src/statement.rs#L111

?

Then you can intercept all the queries.

Reply:
Seems a good idea. Let me have a try.